number/src/lib.rs

```diff
@@ -31,6 +33,60 @@ pub fn log2_exact(n: BigUint) -> Option<usize> {
         .filter(|zeros| n == (BigUint::from(1u32) << zeros))
 }
 
+pub type Columns<F> = Vec<(String, Vec<F>)>;
+
+#[derive(Serialize, Deserialize)]
+/// Like Columns, but each column can exist in multiple sizes
+pub struct VariablySizedColumns<F> {
```

Comment:
I don't think this should be inside `number`

Reply:
Hmm, do you have a suggestion?

Of the common dependencies of `backend` and `executor`, for example, it seems like the best fit:

<img width="988" alt="Screenshot 2024-07-09 at 17 32 16" src="https://github.com/powdr-labs/powdr/assets/1701960/0174f137-d740-442e-94db-a4f8deebe641">

Also, this crate handles the serialization (which arguably it shouldn't), so I think it kind of fits.

Reply:
It's much more than just "number". Either we create a new crate or rename this one.

Reply:
why not put it inside the executor? Isn't it one of its main outputs?

Reply:
True, changed it!